.github/workflows/ci.yml

```diff
@@ -19,4 +19,5 @@ jobs:
           cache: 'npm'
           node-version: '14'
       - run: npm ci
+      - run: npx percy exec -- gatsby build
       - run: npx gatsby build
```

Comment:
Is this still needed? I assume the previous line does all the things needed for Gatsby already?